pkg/snapshot/store.go

```diff
@@ -788,8 +797,16 @@ func GetGlobalStore(ctx context.Context, kotsadmNamepsace string, kotsadmVeleroB
 
 func mapAWSBackupStorageLocationToStore(kotsadmVeleroBackendStorageLocation *velerov1.BackupStorageLocation, store *types.Store) error {
 	endpoint, isS3Compatible := kotsadmVeleroBackendStorageLocation.Spec.Config["s3Url"]
-
-	if !isS3Compatible {
+	var domainName string
+	u, err := url.Parse(endpoint)
+	if err == nil {
```

Comment:
the endpoint must be a valid url, so i suggest just returning an error here if `err` is not nil